src/main/java/edu/cornell/kfs/module/purap/businessobject/JaggaerRoleLinkMapping.java

```diff
@@ -0,0 +1,62 @@
+package edu.cornell.kfs.module.purap.businessobject;
+
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+
+public class JaggaerRoleLinkMapping extends PersistableBusinessObjectBase {
```

Comment:
Since this new BO has an "active" flag on it, you may want to have this class implement the MutableInactivatable interface so that it can take advantage of KualiCo's built-in active-indicator handling.

Reply:
ok